src/teaching-element/Assessment/Feedback.vue

```diff
@@ -49,10 +50,18 @@ export default {
     title() {
       return this.type === 'TR' ? 'Suggested Solution' : 'Solution';
     },
+    allFeedback() {
+      return reduce(this.feedback, (acc, it, key) => {
+        const prefix = rules[this.config.type](key);
+        const content = it;
+        acc.push({ prefix, content });
+        return acc;
+      }, []);
+    },
```

Comment:
Destruct `feedback` and `config` from `this`;
```
const { feedback, config }  = this;
```
Rename `it` to `content` and you can delete `const content = it;`

Maybe you counld inline it all now: 
```js
reduce(feedback, 
  (acc, content, key) => acc.push({ prefix: rules[config.type](key), content }), 
  []
);
```

Reply:
We can just use `lodash/map` so it could look like this:
```js
const { config, feedback } = this;
const rule = rules[config.type];
return map(feedback, (content, key) => ({ prefix: rule(key), content }));
```

Reply:
Very nice. 👍